ble_reflector.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+import time
+import sys
+import subprocess
+from parameters import EEGInfo, SharedParameters
+from Mindwave.mindwave import Datapoint
+
+class RelaxReflector:
+    """Utility to output headset and game data over Bluetooth LE
+    """
+    params = None
+    # last_game_data_time = None
+
+    process = None
+
+    @classmethod
+    def create_reflector(self, params):
```

Comment:
Convention on a `@classmethod` would be to use `cls` instead of `self` to distinguish between the class and the instance.